.github/workflows/flowzone.yml

```
@@ -18,3 +18,25 @@ jobs:
     with:
       docker_images: |
         ghcr.io/balena-io/deploy-to-balena-action
+
+  test-deploy-to-cloud:
+    name: Test deploy to balenaCloud
+    runs-on: ubuntu-latest
+    needs: flowzone
```

I think we would want a condition here to make sure this is not a closed event. Running on a closed event wouldn't be an issue but in general tests that run after a merge or close are pretty useless.
```suggestion
    needs: flowzone
    if: github.event_name == 'pull_request' && github.event.action != 'closed'
```

---

I think the action already only runs on pull_requests?

```
on:
  pull_request:
```

But perhaps just the `github.event.action != 'closed'`?

---

That's correct, I just tend to err on the side of catching unpredicted use cases, like someone adding a new trigger without updating the if conditions. Just checking for not closed is fine.